### PR TITLE
Fix travis tests

### PR DESCRIPTION
**Pull request by isuruf**
Travis-CI was not running `bundle exec rspec` in the correct directory.

@abinashmeher999, can you review?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -48,10 +48,11 @@ script:
   # Build ruby gem
   - gem build symengine.gemspec
   # Install ruby gem
-  - gem install symengine-0.1.0.gem --user --verbose -- -DSymEngine_DIR=$our_install_dir
+  - gem install symengine-0.1.0.gem --install-dir $HOME --verbose -- -DSymEngine_DIR=$our_install_dir
```
Comment (abinashmeher999) [inline .travis.yml]: Looks like the it's not able to find the .so file in OS X.
```
/Users/travis/gems/symengine-0.1.0/lib/symengine.rb:1:in `require': cannot load such file -- symengine/symengine (LoadError)
	from /Users/travis/gems/symengine-0.1.0/lib/symengine.rb:1:in `<top (required)>'
	from /Users/travis/gems/symengine-0.1.0/spec/spec_helper.rb:16:in `require_relative'
```
Are we missing something?

**Comment by certik** [comment]
Why does Ruby 1.9.3 fail?

**Comment by isuruf** [comment]
Not sure. It's quite hard to debug without access to OS X.

**Comment by isuruf** [comment]
I've added the OS X test to allow_failures. I will try to debug this when I get access to OS X.

**Comment by abinashmeher999** [comment]
+1 to merge

**Pull request merged**